### PR TITLE
IInnerList interface was extracted to its own file

### PR DESCRIPTION
**Pull request by ArtemTatarinov**
## Proposed changes
- IInnerList interface was extracted to its own file

## Customer Impact
- No

## Regression? 
- No

## Risk
- Minimal

## Test environment(s) 
- Microsoft Windows [Version 10.0.19043.1165]
- .NET SDK 6.0.100-rc.1.21416.15

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/dotnet/winforms/pull/5439)

### CODE REVIEW

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/dotnet/winforms?pullRequest=5439) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/dotnet/winforms?pullRequest=5439) before we can accept your contribution.<br/><br/>:x: ArtemTatarinov [sign now](https://cla.dotnetfoundation.org/dotnet/winforms?pullRequest=5439)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/dotnet/winforms?pullRequest=5439) it.</sub>

File: src/System.Windows.Forms/src/System/Windows/Forms/ListView.ListViewItemCollection.IInnerList.cs [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+namespace System.Windows.Forms
+{
+    public partial class ListView
+    {
+        public partial class ListViewItemCollection
+        {
+            internal interface IInnerList
+            {
+                int Count { get; }
```
Comment (dreddy-work) [inline src/System.Windows.Forms/src/System/Windows/Forms/ListView.ListViewItemCollection.IInnerList.cs:15]: nit: would be nice to clean and order these ( by type or name).

Comment (RussKie) [reply src/System.Windows.Forms/src/System/Windows/Forms/ListView.ListViewItemCollection.IInnerList.cs:15]: Please follow https://docs.microsoft.com/en-us/archive/blogs/brada/internal-coding-guidelines#28-file-organization
@SergeySmirnov-Akvelon can give you more details.

Comment (ArtemTatarinov) [reply src/System.Windows.Forms/src/System/Windows/Forms/ListView.ListViewItemCollection.IInnerList.cs:15]: Thank you for your comments! The properties and the methods of the interface have been already grouped and sorted by alphabet inside each group, according to the guidelines by the link ("Classes member should be alphabetized, and grouped into sections"). Maybe I should add an empty line between those two groups to make them differ visually?

image

Comment (RussKie) [reply src/System.Windows.Forms/src/System/Windows/Forms/ListView.ListViewItemCollection.IInnerList.cs:15]: Ah, yes, a blank line would certainly help. Thank you

**Review by dreddy-work — APPROVED** [review]

**Pull request merged**